src/actions/breakpoints/syncBreakpoint.js

```diff
@@ -78,6 +87,17 @@ export async function syncClientBreakpoint(
   const { location, astLocation } = pendingBreakpoint;
   const previousLocation = { ...location, sourceId };
 
+  // generatedSource.text, in the case of pretty print or original tab
+  // returns the minified contents, which is why it's used
+  // when creating the "originalText"
+  const generatedSource = getSource(getState(), generatedLocation.sourceId);
+  const text = getTextAtPosition(source.text, location.line, location.column);
+  const originalText = getTextAtPosition(
+    generatedSource.text,
+    actualLocation.line,
+    actualLocation.column
+  );
```

Comment:
this likely should be done closer to where the `createSyncData` call is made. the rationale for that is that the bp could move bc of ast bps and other factors...